server/app/services/CfJsonDocumentContext.java

```diff
@@ -218,6 +219,31 @@ public void putRepeatedEntities(Path path, ImmutableList<String> entityNames) {
     }
   }
 
+  /**
+   * Puts an array at a given path, building parent objects as needed.
+   *
+   * @param path the {@link Path} where the array should be added.
+   * @param entityNames a {@link List} containing service area results.
+   */
+  public void putServiceAreaInclusionEntities(
```

Comment:
Seems like this would live better on ApplicantData since it's referencing scalars specific to that. (And same for read)